bin/q

```diff
@@ -179,7 +182,10 @@ class Sqlite3DB(object):
             self.cursor.execute(q)
             if self.cursor.description is not None:
                 # we decode the column names, so they can be encoded to any output format later on
-                query_column_names = [c[0].decode('utf-8') for c in self.cursor.description]
+                if six.PY2:
+                    query_column_names = [unicode(c[0],'utf-8') for c in self.cursor.description]
+                else:
+                    query_column_names = [c[0] for c in self.cursor.description]
```

Comment:
What about __c[0].decode('uft-8')__ in both Py2 and Py3?

Reply:
the cursor returns byte strings for py2, and already-decoded strings for py3.